frontend/src/components/CommentView.js

```diff
@@ -0,0 +1,275 @@
+import React, { useState } from 'react'
+import { Slate, Editable, withReact } from 'slate-react'
+import { createEditor } from 'slate'
+import { withHistory } from 'slate-history'
+import {
+    Text,
+    Image,
+    Box,
+    Popover,
+    PopoverTrigger,
+    PopoverContent,
+    HStack,
+} from '@chakra-ui/react'
+
+import { Element, Leaf } from './SlateComponents'
+import CommentInput from './CommentInput'
+
+const timeAgo = (dateString) => {
```

Comment:
we should probably move this into a helpers file, as this sort of calculation/formatting might be used elsewhere and shouldn't be specific to this file